### ISSUE TITLE
Run tests with something other than phantom (slimerjs)

### ISSUE THREAD

**Issue by JarrodCTaylor** [reagent-project/reagent-template#72]
I am building a project and utilizing [storage-arom](https://github.com/alandipert/storage-atom) to interface with the browsers localStorage. This works as expected in the app. However, running tests with Phantom is not playing nice [issue #10](https://github.com/alandipert/storage-atom/issues/10).  Looking over the readme for clojurescript.test they recommend using [slimjerjs](https://github.com/cemerick/clojurescript.test#slimerjs-and-phantomjs). According an [issue](https://github.com/cemerick/clojurescript.test/issues/10) in the clojurescript.test repo slimerjs is a drop in replacement for `phantomjs`, but I seem to be missing something with the reagent-template. When I sub in slimerjs I only get the following outout when trying to run tests.

```
Compiling ClojureScript.
Running ClojureScript test: unit
Subprocess failed
```

**Comment by JarrodCTaylor** [comment]
I still haven't had any luck being able to run tests in an app that accesses localStorage.  I am not committed to `slimerjs` being the solution if something else can accomplish the goal. 

I would be willing to help work on a solution if there is a preferred path and if I had an idea of how to start.

**Comment by yogthos** [comment]
It might be worth asking a question on the ClojureScript google groups or Slack to see if anybody had success with this. If you can get it working I can definitely help with finishing up the pr and pushing this out in the new template.

**Comment by JarrodCTaylor** [comment]
Will do. I will let you know if I get anything worked out.

**Comment by JarrodCTaylor** [comment]
After poking around I came across [this issue](https://github.com/cemerick/clojurescript.test/issues/77) turns out the inability to access localStorage was a known issue inside the clojurescript.test project and was fixed in version 0.3.3. I have confirmed that the tests will run as expected when using version 0.3.3 and open PR #75 to update the version in the template.

**Comment by yogthos** [comment]
fantastic!
